scanner/services/validators/validators.go

```diff
@@ -98,13 +98,22 @@ func validateContents(contents *v4.Contents) error {
 	if contents == nil {
 		return nil
 	}
-	if err := validateList(contents.GetPackages(), "Contents.Packages", validatePackage); err != nil {
+	if err := validateMap(contents.GetPackages(), "Contents.Packages", validatePackage); err != nil {
 		return err
 	}
-	if err := validateList(contents.GetDistributions(), "Contents.Distributions", validateDistribution); err != nil {
+	if err := validateList(contents.GetPackagesDEPRECATED(), "Contents.PackagesDEPRECATED", validatePackage); err != nil {
 		return err
 	}
-	if err := validateList(contents.GetRepositories(), "Contents.Repositories", validateRepository); err != nil {
+	if err := validateMap(contents.GetDistributions(), "Contents.Distributions", validateDistribution); err != nil {
+		return err
+	}
+	if err := validateList(contents.GetDistributionsDEPRECATED(), "Contents.DistributionsDEPRECATED", validateDistribution); err != nil {
+		return err
+	}
+	if err := validateMap(contents.GetRepositories(), "Contents.Repositories", validateRepository); err != nil {
+		return err
+	}
+	if err := validateList(contents.GetRepositoriesDEPRECATED(), "Contents.RepositoriesDEPRECATED", validateRepository); err != nil {
 		return err
 	}
 	for k, envs := range contents.GetEnvironments() {
```

Comment:
nit: should `EnvironmentsDEPRECATED` also be validated?

Reply:
oops thanks. Luckily I already added it to the test, but just forgot to add it to the validation...